src/Microsoft.PowerShell.Commands.Utility/commands/utility/ReadConsoleCmdlet.cs

```diff
@@ -55,10 +55,9 @@ public sealed class ReadHostCommand : PSCmdlet
         }
 
         /// <summary>
-        /// Set to no echo the input as is is typed.
+        /// Set to no echo the input as is is typed. If set then the cmdlet returns a secure string.
```

Comment:
To fix CodeFActor issue the comment should be started with "Gets or sets the switch..."